### PR TITLE
App icon

### PR DESCRIPTION
**Pull request by ErikaJZhou**
Add draft app icon.

### CODE REVIEW

**Comment by ErikaJZhou** [comment]
Jules said there's already a .gitignore

File: CityAssistant/CityAssistant.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -12,7 +12,6 @@
 		BD163B852A47A27A0048AE2B /* POIMapView.swift in Sources */ = {isa = PBXBuildFile; fileRef = BD163B842A47A27A0048AE2B /* POIMapView.swift */; };
 		BD163B872A47AE070048AE2B /* POICard.swift in Sources */ = {isa = PBXBuildFile; fileRef = BD163B862A47AE070048AE2B /* POICard.swift */; };
 		BD163B892A47C08F0048AE2B /* LocationManager.swift in Sources */ = {isa = PBXBuildFile; fileRef = BD163B882A47C08F0048AE2B /* LocationManager.swift */; };
-		BD163B8E2A49636D0048AE2B /* LaunchScreen.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = BD163B8C2A49636D0048AE2B /* LaunchScreen.storyboard */; };
```
Comment (jm564) [inline CityAssistant/CityAssistant.xcodeproj/project.pbxproj]: Did you mean to remove this?

Comment (ErikaJZhou) [reply CityAssistant/CityAssistant.xcodeproj/project.pbxproj]: I removed the Storyboards that were in there because we're using SwiftUI.

**Comment by ErikaJZhou** [comment]
LaunchScreen.storyboard is back but the 2D map view and City Tour views seem to be broken and I'm not sure what happened.
City tour view goes to an error page, and 2D map view snaps the view back to the user's location when you try to pan away.

**Review by jm564 — APPROVED** [review]

**Pull request merged**